sample/build.gradle

```diff
@@ -36,18 +36,18 @@ fladle {
     smartFlankGcsPath = "gs://test-lab-yr9w6qsdvy45q-iurp80dm95h8a/flank/test_app_android.xml"
     configs {
         oranges {
-            useOrchestrator = false
-            testTargets = [
+            useOrchestrator.set(false)
+            testTargets.set(project.provider { [
```

Comment:
same here as above and for the rest of the file.

Reply:
https://github.com/runningcode/fladle/pull/176#discussion_r493391055